### PR TITLE
Upgrade gunicorn and use meinheld worker

### PR DESCRIPTION
**Pull request by jgmize**
See also https://github.com/mozmar/lumbergh/pull/194 and https://github.com/mozmar/snippets-service/pull/234

### CODE REVIEW

**Comment by jgmize** [comment]
This has been deployed in https://ci.us-west.moz.works/blue/organizations/jenkins/mdn_multibranch_pipeline/detail/meinheld-demo/7/pipeline to http://mdn-meinheld-demo.virginia.moz.works/en-US/

**Comment by jgmize** [comment]
@stephaniehobson if you'd like to test this locally before it merges, you can check out this branch and then temporarily modify your local docker-compose.yml by adding the tag `:b9a551c` to the end of https://github.com/mozilla/kuma/blob/master/docker-compose.yml#L4 which will cause you to use the image built and pushed by 
https://ci.us-west.moz.works/blue/organizations/jenkins/mdn_multibranch_pipeline/detail/meinheld/5/pipeline when you run `docker-compose up`.

**Comment by metadave** [comment]
+1 from the SRE side of things

File: requirements/default.txt [hunk 1]
```diff
@@ -257,3 +257,32 @@ wheel==0.29.0 \
...
     --hash=sha256:826ffe5d608c9dc8daebef1b0b43d01f7958f17c2fce36e75c80e26160172c4f \
     --hash=sha256:5aea935dfc09ef2beeb76960b4a808b0bbe65e85fb0b0312434b9c365ca02a41
+
+# Fix gunicorn worker timeout problems with default sync worker
+meinheld==0.6.1 \
+    --hash=sha256:40d9dbce0165b2d9142f364d26fd6d59d3682f89d0dfe2117717a8ddad1f4133 \
+    --hash=sha256:293eff4983b7fcbd9134b47706b22189883fe354993bd10163c65869d141e565
+
+# required for meinheld
+greenlet==0.4.12 \
```
Comment (jwhitlock) [inline requirements/default.txt]: Move this to ``requirements/constraints.txt``

**Pull request merged**